src/oscar/templates/oscar/base.html

```diff
@@ -7,6 +7,7 @@
 
         <meta http-equiv="content-type" content="text/html; charset=UTF-8" />
         <meta name="created" content="{% now "jS M Y h:i" %}" />
+        <meta name="title" content="{% block meta_title %}{% endblock %}" />
```

Comment:
This is not a valid element. The title needs to go into the `<title>` tag, for which we already have a `{% block title %}`. The `catalogue/detail.html` and `catalogue/browse.html` templates override this block and we need to adjust the content inside each.